activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+*   Add option to skip joins for associations.
+
+    In a multiple database application, associations can't join across
+    databases. When set, this option tells Rails to make 2 or more queries
+    rather than using joins for associations.
+
+    Set the option on a has many through association:
+
+    ```ruby
+    class Dog
+      has_many :treats, through: :humans, skip_joins: true
```

Comment:
I think `:disable_joins` is more descriptive on the intention of disabling the join in the association table.

Reply:
I like that, will update tomorrow.